docs/user/00-20-overview-api-rule-controller.md

```diff
@@ -0,0 +1,24 @@
+# API Rule Controller
+
+## Overview
+
+API Rule Controller is part of Kyma API Gateway Operator. It uses [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper) and [Istio Service Mesh](https://istio.io/) resources to expose and secure APIs. The controller is responsible for:
```

Comment:
```suggestion
API Rule Controller is part of Kyma API Gateway Operator. It uses [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper) and [Istio Service Mesh](https://istio.io/) resources to expose and secure APIs.
```
I'd remove this point because we already mention the controller's responsibility in the previous sentence